.github/CODEOWNERS

```diff
@@ -632,6 +632,7 @@
 /tools/ebpf/                            @DataDog/ebpf-platform
 /tools/gdb/                             @DataDog/agent-shared-components
 /tools/go-update/                       @DataDog/agent-shared-components
+/tools/NamedPipeCmd/                    @DataDog/windows-kernel-integrations
```

Comment:
Can you also change the owners to WKIT for these files: 
```
pkg/process/net/common_windows.go
pkg/process/net/windows_pipe.go
pkg/process/net/windows_pipe_testutil.go
```

Reply:
common_windows is very generic, I think it should be Windows agent in general.
The other ones I will change to WKIT.

Reply:
Those files are still showing up for me to review. Maybe the order of the entries matters? i.e. the specific ones should go before the broader one (/pkg/process/net/)?

Reply:
I think that is because you cannot change the codeowner for review purposes in a PR. The previous codeowner has to approve the change.